src/ExactOnline.Client.Sdk/Models/EolResponseHeader.cs

```diff
@@ -6,5 +6,5 @@ public class EolResponseHeader
 	/// HTTP response header to indicate the shaping / throttling rate limit
 	/// more info: https://support.exactonline.com/community/s/knowledge-base#All-All-DNO-Simulation-gen-apilimits
 	/// </summary>
-	public RateLimit RateLimit { get; set; }
+	public RateLimit RateLimit { get; set; } = default!;
```

Comment:
Is this safe?

Reply:
Well, it's never being created without providing a value for RateLimit... but we should probably make that whole EolResponseHeader object graph readonly.